### PR TITLE
Documents the MODULE command(s)

### PR DESCRIPTION
**Pull request by itamarhaber**

### CODE REVIEW

**Comment by itamarhaber** [comment]
@stockholmux @bmansheim please review this.

File: commands/module-list.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Returns information about the modules loaded to the server.
+
+@return
+
+@array-reply: list of loaded modules. Each element in the list represents a
```
Comment (stockholmux) [inline commands/module-list.md]: Not a big deal, but I would probably avoid the term 'list' as it can be confused for the data structure.

File: commands/module-load.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Loads a module from a dynamic library at runtime.
+
+This command loads and initializes the Redis module from the dynamic library
+specified by the `path` argument. The `path` should be the absolute path,
+including the full filename, of the library. Any additional arguments are passed
```
Comment (stockholmux) [inline commands/module-load.md]: I'd reword this. The comma separated aside is awkward and gets in the way of understanding. One solution: If you move the aside to the end, I think it makes more sense.

File: commands/module-unload.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Unloads a module.
+
+This command unloads the module specified by `name`. Note that the module's name
+is reported by the `MODULE LIST` command, and may differ from the dynamic
+library's filename.
+
+Known limitations:
+
+*   Modules that register custom data types can not be unloaded.
```
Comment (stockholmux) [inline commands/module-unload.md]: IIRC, If you load a module, write some keys with the module data type then restart the server, it will choke trying to restore from RDB because of the missing data types. Should this be in the limitations?

File: commands/module-list.md (hunk 1 above)
Comment (itamarhaber) [reply commands/module-list.md]: As it already in the documentation in other commands at the same place (see `HGETALL`). I'd rather stick with the "standard" than be "technically correct" (debest kind of!)

File: commands/module-load.md (hunk 2 above)
Comment (itamarhaber) [reply commands/module-load.md]: Agreed.

File: commands/module-unload.md (hunk 3 above)
Comment (itamarhaber) [reply commands/module-unload.md]: I don't think so... you can't load an RDB after unloading a module (unless you use DEBUG). That said, I believe that limitation is buried somewhere in the modules' docs (need to find ref though). In any case, once https://github.com/antirez/redis/pull/5926 is released we'll need to update this.

Comment (stockholmux) [reply commands/module-unload.md]: @itamarhaber fair enough. I'm pretty sure that limitation exists if you stop the redis server, change the redis.conf then restart it again. But, I suppose that is a different situation than `MODULE UNLOAD` (which I rarely use)
